x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plan/physical/FragmentExec.java

```diff
@@ -25,13 +25,13 @@ public class FragmentExec extends LeafExec implements EstimatesRowSize {
      * Estimate of the number of bytes that'll be loaded per position before
      * the stream of pages is consumed.
      */
-    private final Integer estimatedRowSize;
+    private final int estimatedRowSize;
```

Comment:
int is a better default that prevents subtle conversion errors when trying to convert the null Integer to `int`.